code/addons/viewport/src/Tool.tsx

```diff
@@ -112,12 +102,12 @@ interface ViewportToolState {
 }
 
 const getStyles = (
-  prevStyles: ViewportStyles,
-  styles: Styles,
-  isRotated: boolean
-): ViewportStyles => {
-  if (styles === null) {
-    return null;
+  prevStyles?: ViewportStyles,
+  styles?: Styles,
+  isRotated = true
+): ViewportStyles | undefined => {
+  if (!styles || !prevStyles) {
+    return undefined;
   }
```

Comment:
I would prefer here making the parameters not optional. But only annotating prevStyles: ViewPortStyles | undefined. The parameters are always given, it is just that the first parameter be undefined.